chameleon-server/src/main/java/com/chameleonvision/common/vision/frame/provider/FileFrameProvider.java

```diff
@@ -34,21 +40,32 @@ public FileFrameProvider(Path path) {
     * Instantiates a new File frame provider.
     *
     * @param pathAsString The path of the image to read from as a string.
+    * @param fov The fov of the image.
     */
-    public FileFrameProvider(String pathAsString) {
-        this(Paths.get(pathAsString));
+    public FileFrameProvider(String pathAsString, double fov) {
+        this(Paths.get(pathAsString), fov);
     }
 
     private void loadImage() {
         Mat image = Imgcodecs.imread(m_path.toString());
 
         if (image.cols() > 0 && image.rows() > 0) {
+            m_properties = new FrameStaticProperties(image.width(), image.height(), m_fov);
             m_frame = new Frame(image);
         } else {
             throw new RuntimeException("Image loading failed!");
         }
     }
 
+    /**
+    * Set the fov of the image.
+    *
+    * @param fov The fov of the image.
+    */
+    public void setFov(double fov) {
```

Comment:
This is not necessary for File images, but for other Providers, updating the FOV must also force an update to the FrameStaticProperties. I suggest omitting this for now.